global-setup.e2e.ts

```diff
@@ -8,22 +8,29 @@ const localRegistryNxTarget = '@code-pushup/cli-source:local-registry';
 
 export async function setup() {
   await globalSetup();
-  await startLocalRegistry();
-  execSync('npm install -D @code-pushup/cli@e2e');
-  execSync('npm install -D @code-pushup/create-cli@e2e');
-  execSync('npm install -D @code-pushup/nx-plugin@e2e');
-  execSync('npm install -D @code-pushup/eslint-plugin@e2e');
-  execSync('npm install -D @code-pushup/coverage-plugin@e2e');
-  await setupTestFolder('tmp/e2e');
+  try {
+    await setupTestFolder('tmp/local-registry');
+    await startLocalRegistry({ localRegistryTarget: localRegistryNxTarget });
+    console.info('Installing packages');
+    execFileSync(
+      'npx',
+      ['nx', 'run-many', '--targets=npm-install', '--parallel=1'],
+      { env: process.env, stdio: 'inherit', shell: true },
+    );
+    await setupTestFolder('tmp/e2e');
+  } catch (error) {
+    console.info('setup error: ' + error.message);
+  }
```

Comment:
Is this `try`/`catch` appropriate? It will proceed with running tests if the setup fails, that's probably not what we want, right?